.ci/dockerfiles/Dockerfile.xpu

```
@@ -0,0 +1,134 @@
+    ninja install; \
+    test -f /opt/nvidia/nvda_nixl/lib/x86_64-linux-gnu/libnixl.so
+
+# Set env variables to capture libs and python packages from default installation path
+RUN if [ -d /opt/nvidia/nvda_nixl/lib/python3/dist-packages/nixl_cu12 ]; then \
+        ln -sfn /opt/nvidia/nvda_nixl/lib/python3/dist-packages/nixl_cu12 /opt/nvidia/nvda_nixl/lib/python3/dist-packages/nixl; \
+    fi
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/nvidia/nvda_nixl/lib/x86_64-linux-gnu/
+ENV PYTHONPATH=$PYTHONPATH:/opt/nvidia/nvda_nixl/lib/python3/dist-packages
+
+# Install pypi dependencies
+RUN pip install bigdl-core==2.4.0b2
```

Hadolint (2.14.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)